utils/filter.c

```diff
@@ -249,6 +251,11 @@ static void add_arg_spec(struct list_head *arg_list, struct uftrace_arg_spec *ar
  */
 void update_trigger(struct uftrace_filter *filter, struct uftrace_trigger *tr, bool exact_match)
 {
+	if (tr->flags & TRIGGER_FL_CLEAR) {
+		filter->trigger.flags &= ~tr->flags;
```

Comment:
Hmm.. what's the intended use case?  I think we can simply do `filter->triggers.flags = 0`, right?

Reply:
I only want to remove a given filter type. E.g. if you have a filter and an trigger set for a symbol, you can remove either the filter or the trigger (triggers are not implemented yet in the agent). Setting the flags to `0` clears all filters/triggers/args.

The following example would preserve the depth trigger on `func`.
```
$ uftrace record -F func -T func@depth=1
$ uftrace -p <pid> -F func@clear
```

Reply:
Hmm.. ok.  Then `-T func@clear` would clear them all.